src/sentry/api/endpoints/group_details.py

```diff
@@ -364,7 +367,9 @@ def put(self, request, group):
                 environment_func=self._get_environment_func(request, group.project.organization_id)
             ),
         )
-
+        metrics.incr(
```

Comment:
I think for these metrics to measure status successfully you'll need to wrap the entire thing in a try/except or use a decorator. Cause if this errors outside the earlier try/except there won't be a incremented metric.